README.md

```diff
@@ -2,7 +2,7 @@
 
 > Cyberpunk HyperTerm theme
 
-![](https://cdn.rawgit.com/staltz/hyperpunk/master/screenshot.png)
+![](screenshot.png)
```

Comment:
We need to keep it how it was before so the screenshot shows also in the npm page.